include/rocksdb/write_batch.h

```diff
@@ -276,6 +276,8 @@ class WriteBatch : public WriteBatchBase {
     virtual bool WriteBeforePrepare() const { return false; }
   };
   Status Iterate(Handler* handler) const;
+  class Iterator;
+  Iterator* NewIterator() const { return new Iterator(rep_); }
```

Comment:
Is it possible to use the `WriteBatch::Iterate` API instead of introducing iterator? I want to reduce rebase overhead when upgrading rocksdb.